pkg/iss/iss.go

```diff
@@ -653,7 +653,7 @@ func (iss *ISS) initOrderers() *events.EventList {
 		// Create segment.
 		seg := &orderers.Segment{
 			Leader:     leader,
-			Membership: maputil.GetSortedKeys(iss.epoch.nodeIDs),
+			Membership: iss.epoch.Membership,
```

Comment:
Yes, because we now store the whole membership in the segment, not just the IDs of the nodes. This is because we need to use this information when instantiating the checkpoint orderer instance.